packages/x-components/src/x-modules/search-box/components/search-input.vue

```diff
@@ -5,6 +5,8 @@
     @click="emitUserClickedSearchBox"
     @focus="emitUserFocusedSearchBox"
     @input="emitUserIsTypingAQueryEvents"
+    @mouseover="emitUserHoveringOverSearchBox"
+    @mouseout="emitUserHoveringOutOfSearchBox"
```

Comment:
I think we should listen for the mouse events on the label inside of the `search-input-placeholder.vue` and not here.